# Check for floor() instead of sqrt() to decide if we need libm
#
# This patch needs to be evaluated to see if it needs to be passed upstream.

--- w3m-0.5.3/configure.ac.orig	2015-02-16 19:03:26.191283591 -0800
+++ w3m-0.5.3/configure.ac	2015-02-16 19:08:11.367778846 -0800
@@ -91,7 +91,7 @@
 AC_W3M_SSL_DIGEST_AUTH
 
 dnl Checks for libraries.
-AC_CHECK_FUNC(sqrt,,[AC_CHECK_LIB(m, sqrt)])
+AC_CHECK_FUNC(floor,,[AC_CHECK_LIB(m, floor)])
 if test x"$enable_mouse" = xyes; then
  AC_SUBST(USE_W3M)
  AC_CHECK_LIB(gpm, Gpm_Open, [AC_DEFINE(USE_GPM)
